extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -202,4 +202,70 @@ describe.configure().retryOnSaucelabs().run('integration amp-bind', function() {
     });
   });
 
+  describe('amp-video integration', () => {
+    it('should change src when the src attribute binding changes', () => {
+      const button = fixture.doc.getElementById('changeVidSrcButton');
+      const vid = fixture.doc.getElementById('video');
+      expect(vid.getAttribute('src')).to
+          .equal('https://www.google.com/unbound.webm');
+      button.click();
+      return waitForBindApplication().then(() => {
+        expect(vid.getAttribute('src')).to
+            .equal('https://www.google.com/bound.webm');
+      });
+    });
+
+    it('should NOT change src when new value is a blocked URL', () => {
+      const button = fixture.doc.getElementById('disallowedVidUrlButton');
+      const vid = fixture.doc.getElementById('video');
+      expect(vid.getAttribute('src')).to
+          .equal('https://www.google.com/unbound.webm');;
+      button.click();
+      return waitForBindApplication().then(() => {
+        expect(vid.getAttribute('src')).to
+            .equal('https://www.google.com/unbound.webm');
+      });
+    });
+
+    it('should NOT change src when new value uses an invalid protocol', () => {
+      const button = fixture.doc.getElementById('httpVidSrcButton');
+      const vid = fixture.doc.getElementById('video');
+      expect(vid.getAttribute('src')).to
+          .equal('https://www.google.com/unbound.webm');
+      button.click();
+      return waitForBindApplication().then(() => {
+      // Only HTTPS is allowed
```

Comment:
Overflow is at +4. Will fix the comment on [this PR](https://github.com/ampproject/amphtml/pull/8010)